lib/web_ui/lib/src/engine/view_embedder/dom_manager.dart

```diff
@@ -4,8 +4,25 @@
 
 import 'package:ui/ui.dart' as ui;
 
+import '../display.dart';
 import '../dom.dart';
 import '../embedder.dart';
+import '../global_styles.dart';
+
+/// The tag name for the Flutter View root element.
+const String kFlutterViewTagName = 'flutter-view';
```

Comment:
Should these be `static const` of `DomManager` (so they're consistent with the ones added in `StyleManager`?)

Also:

![Drop the "The". Just "Facebook". It's cleaner.](https://media.tenor.com/gf5KOGiJ2s0AAAAC/k22p01.gif)

Drop the `k`. Just `flutterViewTagName`. It's cleaner. 😝

Reply:
I like it!